tests/unit/lms/services/course_test.py

```diff
@@ -178,6 +246,16 @@ def svc(self, db_session, application_instance, grouping_service):
             grouping_service=grouping_service,
         )
 
+    @pytest.fixture
+    def get_by_context_id(self, svc):
+        with patch.object(svc, "get_by_context_id") as update_application_instance:
+            yield update_application_instance
+
+    @pytest.fixture
+    def upsert_course(self, svc):
+        with patch.object(svc, "upsert_course") as update_application_instance:
+            yield update_application_instance
```

Comment:
```suggestion
    @pytest.fixture
    def get_by_context_id(self, svc):
        with patch.object(svc, "get_by_context_id") as get_by_context_id:
            yield get_by_context_id

    @pytest.fixture
    def upsert_course(self, svc):
        with patch.object(svc, "upsert_course") as upsert_course:
            yield upsert_course
```

Some copy paste here maybe?